doc/source/python-tiledbsoma-logging.rst

```diff
@@ -0,0 +1,19 @@
+The tiledbsoma.logging module
+=====================
+
+.. currentmodule:: tiledbsoma.logging
+
+.. automodule:: tiledbsoma.logging
+
+
+Functions
+-------
+
+.. autosummary::
+    :toctree: _autosummary/
+    :nosignatures:
+
+    tiledbsoma.logging.debug
+    tiledbsoma.logging.info
+    tiledbsoma.logging.log_io
+    tiledbsoma.logging.warning
```

Comment:
```suggestion
    tiledbsoma.logging.info
    tiledbsoma.logging.debug
    tiledbsoma.logging.warning
    tiledbsoma.logging.log_io
```

the first three are by increasing verbosity (which is not alphabetical)

the fourth one, we should (on this PR or another) mark as non-public b/c it's only meant for use by `tiledbsoma.io`

Reply:
I removed it from the doc. Another pro of having explicitly declared methods!